api/controllers/game/resolve-three.js

```diff
@@ -0,0 +1,66 @@
+module.exports = function (req, res) {
+  const promiseGame = gameService.findGame({gameId: req.session.game});
+  const promisePlayer = userService.findUser({userId: req.session.usr});
+  const promiseCard = cardService.findCard({cardId: req.body.cardId});
+  Promise.all([promiseGame, promisePlayer, promiseCard])
+    .then(function changeAndSave(values) {
+      const [game, player, card] = values;
+      const gameUpdates = {
+        oneOff: null,
+        resolving: null,
+        passes: 0,
+        turn: game.turn + 1,
+        log: [
+          ...game.log,
+          `${userService.truncateEmail(player.email)} took the ${card.name} from the Scrap pile to their hand.`,
+        ],
+        lastEvent: {
+          change: 'resolveThree',
+        },
+      };
+      const updatePromises = [
+        // Update game
+        Game.updateOne(game.id)
+          .set(gameUpdates),
+        // Scrap the three that just resolved
+        Game.addToCollection(game.id, 'scrap')
+          .members([game.oneOff.id]),
+        // Return selected card to player's hand
+        User.addToCollection(player.id, 'hand')
+          .members([card.id]),
+        // Remove selected card from scrap
+        Game.removeFromCollection(game.id, 'scrap')
+          .members([card.id]),
+        // Clear player's frozenId
+        User.updateOne(player.id)
+          .set({
+            frozenId: null,
+          }),
```

Comment:
If we always clear the frozenId in the resolve action, then we can skip setting it in the individual effect resolutions for 3's, 4's and 7's